src/v1/sources/shopify/transformV1.js

```diff
@@ -17,7 +17,7 @@ const processV1Events = async (event) => {
       return processIdentifierEvent(event);
     }
     // handle events from the app pixel.
-    const pixelWebEventResponse = processPixelWebEvents(event);
+    const pixelWebEventResponse = await processPixelWebEvents(event);
```

Comment:
what got changes to add `await` ?

Reply:
`const redisData = await RedisDB.getVal(`pixel:${message.anonymousId}`);`
The user id enrichment call to redis is an asynchronous operation. The parent functions need to be marked async to use await and properly handle the Promise chain.
Any function that calls `processPixelWebEvents` would also need to be async and use await if it needs to wait for the pixel event processing to complete.